flw/src/main/java/org/motechproject/nms/flw/service/impl/FrontLineWorkerImportServiceImpl.java

```diff
@@ -131,13 +132,26 @@ private FrontLineWorker flwFromRecord(Map<String, Object> record, State state) {
             // if the mcts id has changed.
             if (flw != null && mctsFlwId != null && !mctsFlwId.equals(flw.getMctsFlwId())) {
                 throw new CsvImportDataException(String.format("Existing FLW with same MSISDN (%s) but " +
-                                        "different MCTS ID (%s != %s)", msisdn, mctsFlwId, flw.getMctsFlwId()));
+                                        "different MCTS ID (%s != %s)", obscureNumber(msisdn), mctsFlwId, flw.getMctsFlwId()));
             }
         }
 
         return flw;
     }
 
+    // obscure code from LogHelper in API package. Copy-paste since we cannot do a circular reference
```

Comment:
It is used more extensively in api than flw and that would be a stop gap anyways, without 'common' module that other modules can use.